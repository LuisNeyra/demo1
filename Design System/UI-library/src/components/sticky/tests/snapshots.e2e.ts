import { snapIt } from '../../../test';

const component = 'sticky';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap(
      'sticky no scroll',
      `<blaze-sticky top="20">
        <blaze-alert open dismissible type="info">Sticky alert</blaze-alert>
       </blaze-sticky>`
    );
  });
});
