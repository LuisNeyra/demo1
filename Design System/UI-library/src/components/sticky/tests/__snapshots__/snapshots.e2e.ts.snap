// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sticky renders sticky no scroll 1`] = `"<blaze-sticky top=\\"20\\" class=\\"hydrated\\"><!----><div><div> <blaze-alert open=\\"\\" dismissible=\\"\\" type=\\"info\\" class=\\"hydrated\\"><!----><div role=\\"alert\\" class=\\"c-alert c-alert--info\\"><button class=\\"c-button c-button--close\\">Ã—</button>Sticky alert</div></blaze-alert> </div></div></blaze-sticky>"`;
